src/app/signalen/_hooks/useAddErrorCountToPageTitle.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+
+export const useAddErrorCountToPageTitle = (errors: { id: string; label: string }[]) => {
```

Comment:
I think this might be something for more generic Forms docs. It's part of [this 4-pronged approach to tell users about validation errors](https://design-system.service.gov.uk/patterns/validation/#how-to-tell-the-user-about-validation-errors), which is also followed by NLDS.

Reply:
We discussed that we’ll even build this into the component.